ratelimiter/src/main/java/com/flipkart/varadhi/qos/entity/TopicLoadInfo.java

```diff
@@ -0,0 +1,29 @@
+package com.flipkart.varadhi.qos.entity;
+
+
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class TopicLoadInfo {
```

Comment:
Why not use "record class " from java 17? they come inbuilt with tostring as well.

Reply:
Make sense, these should be immutable. Changed to record